maintainers/maintainer-list.nix

```diff
@@ -16424,4 +16424,10 @@
     github = "srounce";
     githubId = 60792;
   };
+  gou4shi1 = {
```

Comment:
Please add the entry in alphabetical (as stated in the file header) order to the list. The entries at the end of the file usually sneaked in when time is an issue.